app/views/ranks/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Listing Ranks</h1>
+
+<table>
+	<thead>
+		<tr>
+			<th>UserID</th>
+			<th>Score</th>
+			<th colspan="3"></th>
+		</tr>
+	</thead>
+
+	<div id="ranking">
```

Comment:
Seid ihr sicher, dass ihr das folgende **in** die Tabelle schreiben wollt?
Braucht ihr diese Tabelle drum herum überhaupt noch? :/